airflow/api_connexion/openapi/v1.yaml

```diff
@@ -3121,6 +3123,22 @@ components:
           readOnly: true
           nullable: true
 
+    DagProcessorStatus:
+      type: object
+      description: |
+        The status and the latest dag processor heartbeat.
+
+        *New in version 2.6.3*
```

Comment:
Is it in 2.6.3?

Reply:
I saw triggerer status added in 2.6.2 and thought it doesn't need to wait till 2.7.0. This doesn't seem to be in 2.6.3rc candidate. I am not sure if this was cherry picked or I missed the release/RC window. cc: @ephraimbuddy